data/store/store.go

```diff
@@ -32,6 +30,7 @@ type DataSetRepository interface {
 
 	GetDataSetsForUserByID(ctx context.Context, userID string, filter *Filter, pagination *page.Pagination) ([]*upload.Upload, error)
 	GetDataSetByID(ctx context.Context, dataSetID string) (*upload.Upload, error)
+	IsDataSetAutomated(ctx context.Context, dataSetID string) (bool, error)
```

Comment:
I'm not entirely sure this required. You can use `GetDataSetById` and add a method `IsContinuous()` on the struct instead of adding this to the repository.

Reply:
Added `IsContinuous()` to `Upload` type.